docker/start.py

```diff
@@ -205,36 +205,43 @@ def main(args, environ):
             config_dir, config_path, environ, ownership
         )
 
-    if mode is not None:
-        error("Unknown execution mode '%s'" % (mode,))
+    args = ["python"] + args[1:]
```

Comment:
I don't think it is a good idea to conflate the "mode" and the synapse parameters in this way.  I suggest adding an explicit `mode` such as `run` which must be specified before any synapse parameters.

Reply:
Yes, also thought of this. Can change that.